hope/presskit.html

```diff
@@ -9,17 +9,16 @@
   <h3>Description</h3>
 
   <p>
-    <b>Hope: A Sky Full of Ghosts</b> is a minimalist 2D sci-fi point-and-click adventure focused on narrative.
-    It draws inspiration from the classic point-and-click games from the early 90's but blended with storytelling
-    elements from modern games. You must overcome challenges through mini-games and puzzles and the decisions you
-    make will shape the course of the story.
+    <b>Hope: A Sky Full of Ghosts</b> is a sci-fi point and click adventure where you captain a disjointed crew on a stolen spaceship in the hopes of surviving NexxusCorp's tyranny
```

Comment:
typo: We're missing a period at the end of this sentence (this is HTML so it doesn't recognize the new lines unless you do `<br/>`).